src/main/java/core/basesyntax/service/operation/ReturnOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public int getQuantity(int defaultQuantity, int fruitQuantity) {
+        return defaultQuantity + fruitQuantity;
```

Comment:
The parameter names `defaultQuantity` and `fruitQuantity` are not intuitive. Consider renaming them to better reflect their purpose, such as `currentQuantity` and `returnQuantity`.